packages/grid/x-data-grid/src/components/cell/GridActionsCellItem.tsx

```diff
@@ -8,6 +8,7 @@ import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
 export type GridActionsCellItemProps = {
   label: string;
   icon?: React.ReactElement;
+  component?: React.ElementType;
```

Comment:
How about adding a comment to link it to where it comes from
```suggestion
  // https://mui.com/material-ui/api/button-base/#ButtonBase-prop-component
  component?: React.ElementType;
```